frontend/package.json

```diff
@@ -9,11 +9,9 @@
     "web": "expo start --web",
     "prettier-format": "prettier --config .prettierrc 'src/**/*.js' --write",
     "lint:fix": "npx eslint . && npm run prettier-format"
-
   },
   "dependencies": {
     "@expo/vector-icons": "^14.0.0",
-    "@react-navigation/native": "^6.1.10",
```

Comment:
This was in main previous to your branch and presumably is used somewhere. Easiest thing to do is run `npm install @react-navigation/native` to reinstall it.